### DISCUSSION TITLE
Is there a way to speed up video

### DISCUSSION THREAD

**Discussion by Lioncub901** [keiwando/toonsquid#671] [Q&A] [answered]
I sometimes like to use ToonSquid to edit videos. I was wondering is it possible to speed up videos?

**Comment by kavlor** [comment]
You can increase the frame rate if you mean like video clips by tapping the fps just above the time line , audio might get desynced and it will give you a warning.

There is a timing aspect when you go to the pull out tab on the right , you can keyframe up to frame 100 although you can tap it and put higher numbers but it will flick back to 1-100 if you use the slider.

**Reply by Lioncub901** [reply, marked as answer]
I figured out that you can use the time property to speed up time through key framing

**Reply by kavlor** [reply]
Great
